engine/gamesys/src/gamesys/components/comp_mesh.cpp

```diff
@@ -50,7 +50,7 @@ namespace dmGameSystem
         Matrix4                         m_Local;
         Matrix4                         m_World;
         uint32_t                        m_MixedHash;
-        CompRenderConstants             m_RenderConstants;
+        HComponentRenderConstants       m_RenderConstants;
```

Comment:
All our components should use this way with the render constants.
(We'll update the rest later on)